src/styles/blocks/card.scss

```
@@ -0,0 +1,116 @@
+
+.card {
+  box-sizing: border-box;
+  font-family: $font-family;
+  font-style: normal;
+  width: $width-card;
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+  margin-right: $gap-between-card;
+  background: $backgroundcolor-page;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__foto {
+    height: 134px;
+    margin: auto;
+  }
+
+  &__item-model {
+    margin-top: 40px;
+    font-weight: $font-weight-normal;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-font-bold;
+    text-decoration: none;
+  }
+
+  &__item-kod {
+    font-weight: $font-weight-light;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-font-normal;
+    margin-top: 4px;
+  }
+
+  &__feedback {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__stars {
+    display: flex;
+
+    @for $index from 1 through 5 {
+      &--amount-stars--#{$index} :nth-of-type(-n+#{$index}) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background: url(../images/star.svg) no-repeat center;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__feedback-count {
+    font-weight: $font-weight-light;
+    font-size: 10px;
+    color: $color-font-bold;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+  }
+
+  &__price-word {
+    font-weight: $font-weight-light;
+    font-size: 12px;
+    color: $color-font-normal;
+  }
+
+  &__uah {
+    font-weight: $font-weight-bold;
+    font-size: 16px;
+    line-height: 18px;
+    color: $color-font-normal;
+  }
+
+  &__buy {
+    height: 40px;
+    width: 166px;
+    background: $colore-blue;
+    border: 1px solid $colore-blue;
+    border-radius: 5px;
+    margin-top: 16px;
+    text-transform: uppercase;
+    font-family: $font-family;
+    font-weight: $font-weight-bold;
+    font-size: 14px;
+    color: $backgroundcolor-page;
+
+    &:hover {
+      background: $backgroundcolor-page;
+      border: 1px solid $colore-blue;
```

Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover
```suggestion
      border-color: $colore-blue;
```

---

I fixed